core/src/cats/drop_bareos_tables.in

```diff
@@ -31,109 +31,55 @@
 # change to root dir
 cd /
 
+. "@scriptdir@"/bareos-config-lib.sh
 
-. @scriptdir@/bareos-config-lib.sh
-
-db_name="${db_name:-`get_database_name @db_name@`}"
-db_user="${db_user:-`get_database_user @db_user@`}"
-db_version=`get_database_version`
-bareos_sql_ddl=`get_database_ddl_dir`
+db_type=$(get_database_driver_default)
+db_name="${db_name:-$(get_database_name @db_name@)}"
+bareos_sql_ddl="$(get_database_ddl_dir)"
 temp_sql_schema="/tmp/drops.sql.$$"
-default_db_type=`get_database_driver_default`
-working_dir=`get_working_dir`
 
-#
-# See if the first argument is a valid backend name.
-# If so the user overrides the default database backend.
-#
+# Check if some parameters are given, try to detect old usage with db_type
 if [ $# -gt 0 ]; then
-   case $1 in
-      sqlite3)
-         db_type=$1
-         shift
-         ;;
-      mysql)
-         db_type=$1
-         shift
-         ;;
+  case "$1" in
       postgresql)
-         db_type=$1
+         warn "Parameter db_type is no more needed"
          shift
          ;;
       *)
+         error "db_type $1 is not more supported, only postgresql"
```

Comment:
The above lines are the same in all  scripts as it seems, don't we want to put them into one function in the lib?

Reply:
That's seems to be a good idea, They were there and I forgot to imagine this, my first idea was to simply remove all, and don't be nice with older calls.